llvm/test/MC/Sparc/sparc-vis.s

```diff
@@ -1,4 +1,237 @@
-! RUN: llvm-mc %s -triple=sparcv9 -mcpu=niagara -show-encoding | FileCheck %s
+! RUN: not llvm-mc %s -triple=sparcv9 -show-encoding 2>&1 | FileCheck %s --check-prefixes=NO-VIS
```

Comment:
might want `--implicit-check-not=error:` to ensure that you do not miss any error check pattern.